test/FastTests/Client/CRUD.cs

```diff
@@ -14,7 +14,7 @@ public CRUD(ITestOutputHelper output) : base(output)
         {
         }
 
-        [RavenTheory(RavenTestCategory.ClientApi)]
+        [RavenTheory(RavenTestCategory.ClientApi | RavenTestCategory.Conventions)]
```

Comment:
those should not be marked with Conventions, for us 'Conventions' means convention tests like 'check if someone added XYZ when developing a feature', not DocumentConvention tests